goagen/codegen/finalizer_test.go

```diff
@@ -89,4 +107,8 @@ if ut.Foo == nil {
 	hashAssignmentCode = `if ut.Foo == nil {
 	ut.Foo = map[string]string{"bar": "baz"}
 }`
+	datetimeAssignmentCode = `var defaultFoo, _ = time.Parse(time.RFC3339, "1978-06-30T10:00:00+09:00")
```

Comment:
`goagen` validated the format of the date time string , so genarated code discards the error of time.Parse().